backend/playlist/utils/config/abbreviation.yaml

```diff
@@ -262,6 +264,7 @@ ENV SCI:
 
 ETH STD:
     - "Ethnic Studies"
+    - "Chicanx Latinx Studies"
```

Comment:
I think this should go under CHICANO. [Here](https://classes.berkeley.edu/content/2022-fall-chicano-39-001-sem-001) is an example of one of the courses that appears with this course subject.